src/coreclr/inc/corcompile.h

```diff
@@ -185,6 +185,22 @@ inline BOOL CORCOMPILE_IS_FIXUP_TAGGED(SIZE_T fixup, PTR_CORCOMPILE_IMPORT_SECTI
         PCODE                m_pTarget;
     };
 
+#elif defined(TARGET_LOONGARCH64)
+    struct  CORCOMPILE_EXTERNAL_METHOD_THUNK
+    {
+        // Array of words to do the following:
+        // pcaddi   $r21, 0          ; Save the current address relative to which we will get slot ID and address to patch.
+        // ld.d     $r21, $r21, 16   ; Load the target address.
+        // jirl     $r0,$r21,0       ; Jump to the target
+        DWORD m_rgCode[3];
+
+        DWORD m_padding; //aligning stack to 16 bytes
```

Comment:
This structure has nothing to do with stack. Is the comment wrong?

Reply:
> This structure has nothing to do with stack. Is the comment wrong?

Yes.
I should add a macro at here, `prestup.cpp-Line:2363 #elif  defined(TARGET_ARM) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64)`

But I also have a question.
Previous within the .NET6-runtime, the initialing code is in the file `src/coreclr/zap/zapimport.cpp:489`.
But now the zapimport.cpp had been deleted.
Not only the LoongArch64, the others architecturs are also do nothing for initialing the m_rgCode.

Reply:
Yes, `CORCOMPILE_EXTERNAL_METHOD_THUNK` is no longer used. It would be best to delete it instead of trying to fix it up for LoongArch64.

Reply:
OK, I will delete it for LoongArch64.

Reply:
> Yes, `CORCOMPILE_EXTERNAL_METHOD_THUNK` is no longer used. It would be best to delete it instead of trying to fix it up for LoongArch64.

Maybe I have to keep a define for LoongArch64 as the same with other architecture rather than compiling error for undefined value.

Reply:
I am deleting it in #65869

Reply:
> I am deleting it in #65869

OK, I will update it after you merge that.
Thanks